src/components/Main/Members/MembersHeader.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+
+import { fade, makeStyles, withStyles, Theme, createStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableRow from '@material-ui/core/TableRow';
+import IconButton from '@material-ui/core/IconButton';
+import SearchIcon from '@material-ui/icons/Search';
+import InputBase from '@material-ui/core/InputBase';
+
+import AddCircleIcon from '@material-ui/icons/AddCircle';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+export interface MembersHeaderProps {
+    localChurch: string;
+    onSearchChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    handleAddOpen: () => void;
+    handleDeleteOpen: () => void;
+}
+
+export const MembersHeader = ({localChurch, onSearchChange, handleAddOpen, handleDeleteOpen}: MembersHeaderProps) => {
+    const classes = useStyles();
+
+    return (
+        <TableContainer component={Paper}>
+          <Table className={classes.table} aria-label="simple table">
```

Comment:
Either remove this specific aria-label, or change it for screen reader a11y:
```suggestion
          <Table className={classes.table} aria-label="members table controls">
```